magento/inventory
# Remove magento/sales dependency from inventoryBundleProduct module.

### Description (*)


### Fixed Issues (if relevant)

1. none.

### Manual testing scenarios (*)

1. Create bundle product on custom stock.
2. Verify order, invoice and credit memo with bundle product are created successfully

### Questions or comments


### Contribution checklist (*)
